### ISSUE TITLE
Consistency of Entry information pages

### ISSUE THREAD

**Issue by richardofsussex** [FreeUKGen/FreeBMD2#564]
The standard Entry information page, as displayed from search results:

![image](https://github.com/FreeUKGen/FreeBMD2/assets/2197102/eba3add2-476f-47b2-ac40-526535c0a629)

does not look the same as, or contain the same information as, the Entry information page which is displayed when you enter the 'hash URL' (e.g. http://localhost:3000/entry-information/9Llytp0ZnMt2i7KwdA5uZA/hash):

![image](https://github.com/FreeUKGen/FreeBMD2/assets/2197102/0dd7e376-d6f2-4c6e-8d35-c104affe7c6d)

There may be other variants of the Entry information page elsewhere in the BMD2 ecosystem: @Vino-S can advise. I think they should all have the same design and information content, unless there are clear reasons for them to vary. Ideally they should (for ease of maintenance in future) all be generated from the same code.

**Comment by richardofsussex** [comment]
It looks as though there are two variants within `BestGuessHash`: `bmd1_show.html.erb` and `show.html.erb`. as well as the 'default' page which is defined in `BestGuess`; also `show.html.erb`.

**Comment by richardofsussex** [comment]
A single saved record has a similar Entry information display to the standard search results one, but lacks the More information row:

![image](https://github.com/FreeUKGen/FreeBMD2/assets/2197102/a7bfc0ba-ce1b-45e0-9cd4-d31f52220330)

**Comment by KristinaGadzhieva** [comment]
I suggest checking whether the page follows the wireframe design. For example, buttons 'Back to results' and 'Generate citation' should be aligned to the left side of the page.  The fields in the table need to be done according to the wireframe. (We have separate stories for this)

**Comment by nurifreembd** [comment]
consider aligning text left if it helps with the presentation of the records to the Researchers.

**Comment by nurifreembd** [comment]
Consider updating the envelope image within the postem row to read 'Add a postem'.

**Comment by KristinaGadzhieva** [comment]
Nuri, thank you for your comments. I have added a point about the envelope to story #595. 
I can also add the text alignment, @richardofsussex @DeniseColbert, what do you think?

**Comment by KristinaGadzhieva** [comment]
@nurifreembd  Nuri, I'm sorry, I forgot that we have changed the Postems view and have a different implementation on test 3. On test 3 there is 'Postems', and when you click on it, the option 'Add a postem' appears. Could please have a look at test 3?

**Comment by nurifreembd** [comment]
Checked on test3, when you hover over the envelope it reads postems. On
click it reads postems loading then launches the following message

We're sorry, but something went wrong with processing your request. A
report has been produced and sent to the webmaster. Please return to your
prior activity but try to avoid doing the exact same action again.

**Comment by KristinaGadzhieva** [comment]
@nurifreembd I've checked Postems on Beta, works well

**Comment by KristinaGadzhieva** [comment]
Suggestions for discussion:
1. Update a link to the Help section about quarters, districts, and how to find the partner in a marriage.
2. Separate the two links to Help page more clearly. The second one starts with 'Need help?'.
3. Shorten sentences related to Help (to make them easier and quicker to read) 
4. Move 'Previous Result', 'Next result' below the table or at the bottom of the page 
5. Allign the links 'Back to results', 'Genarate citation', 'Print' to the left (to make them consistent with the Results page)

@richardofsussex @DeniseColbert @nurifreembd please, let me know what you think.

**Comment by nurifreembd** [comment]
@KristinaGadzhieva Retested too and it works. Thanks

**Comment by nurifreembd** [comment]
items 1-5 seems good and thank you

**Comment by richardofsussex** [comment]
A general observation: this is _not_ what this story was originally about. It was talking about Entry information pages being displayed in different contexts (e.g. as a saved record), and the fact that these different manifestations of the Entry information page were not identical. It now appears to be a 'fix list' for the principal Entry information page.

**Comment by richardofsussex** [comment]
Having said that ...

1-3. Yes we have effectively the same text about help, twice, with different links in each, neither of which is correct.
4. This has been actioned in beta, but the 'back' and 'forward' links should be on the same 'line'. At present 'forward' is below 'back'.
5. Agreed

**Comment by KristinaGadzhieva** [comment]
Right, Richard. However, we have similar stories related to inconsistencies on other pages. Let's keep this story active as part of the fix list.

**Comment by KristinaGadzhieva** [comment]
1-3   @richardofsussex Richard, could you please help with text adjustments related to the help?
4 -   On Beta navigation buttons are also 'between' text. They are not quite visible. Could we consider moving them under the table?
5- _Agreed 'Allign the links 'Back to results', 'Genarate citation', 'Print' to the left (to make them consistent with the Results page) '_  @Vino-S could you please make it on test 3 ?

**Comment by richardofsussex** [comment]
1-3 Yes, I can update that text and remove the duplication. I'll need to check where the link should be pointing. 
Note that the current help doesn't say anything about Districts, so the link text is misleading.

**Comment by richardofsussex** [comment]
Further suggestions:
6. Mixed case for Record type ("Death" not "DEATH")
7. Work out whether it's Age at Death or Date of birth and output the appropriate heading.

**Comment by richardofsussex** [comment]
@Vino-S please push `fb_564_rbl` to test3. I have attempted to deal with all the issues above (including 5).

**Comment by Vino-S** [comment]
@richardofsussex test3 updated

**Comment by richardofsussex** [comment]
Thanks @Vino-S. 4 hasn't quite worked - if there is a query about the record, it appears above the previous/next record links. I'll fix and re-push my branch.

**Comment by richardofsussex** [comment]
`fb_564_rbl` now updated and pushed to GitHub.

**Comment by Vino-S** [comment]
Hi @richardofsussex , test3 updated

**Comment by KristinaGadzhieva** [comment]
Thank you.
1-3 Let's get back to them after completing the Help page.
4, 5 - Now is fine. 
6, 7 - I agree.

**Comment by KristinaGadzhieva** [comment]
4 -7  Ready for deployment

**Comment by richardofsussex** [comment]
I think you'll find that it is left-aligned in test3. Branch `fb-rbl-604-3` has the code, which should be deployed to beta. 

(This is a bit tricky to spot, because on my system this text flashes up and then instantly disappears on page load. Something to do with whether javascript is enabled, @Vino-S ?)

**Comment by KristinaGadzhieva** [comment]
Yes, sorry and thank you. The text changes are done in #595.

**Comment by Vino-S** [comment]
Deployed

**Comment by KristinaGadzhieva** [comment]
Can we transfer 'No data' and dashes in the table on the Entry information page as well ?

**Comment by richardofsussex** [comment]
We can do 'No data'. Dashes should never be required/relevant in the Entry information page, since only data items which are relevant to the specific event type are displayed.

**Comment by KristinaGadzhieva** [comment]
Right, then we need to transfer 'No data'

**Comment by richardofsussex** [comment]
OK, all otherwise empty cells in Entry information should now say 'No data'. Waiting for @Vino-S to push this to beta.

**Comment by Vino-S** [comment]
Depoloyed

**Comment by KristinaGadzhieva** [comment]
@Vino-S Could you please update the link on the Entry information page to the relevant section of Help page and deploy it on Beta?

**Comment by KristinaGadzhieva** [comment]
@Vino-S a link to the scans also needs to be changed (it opens in a new tab), the screeshot below

<img width="1083" height="773" alt="Image" src="https://github.com/user-attachments/assets/2d4f1719-8319-4468-820c-6b1bf4a41af1" />
